docs/installingNative.md

```diff
@@ -28,6 +130,24 @@ This page describes the procedure to install and run PrairieLearn without any us
   cd PrairieLearn
   ```
 
+!!! note "Setup a venv"
+
+    It is recommended to use a virtual environment for Python dependencies. You can create a virtual environment within PrairieLearn:
+
+    === "Native"
+
+        ```sh
+        python3.10 -m venv venv
+        source venv/bin/activate
+        ```
+
+    === "uv"
+
+        ```sh
+        uv venv
```

Comment:
I'd rather we be explicit about which version of Python we use; it'll be increasingly likely as time goes on that people already have `uv` installed and that it's managing a different version of Python.

```suggestion
        uv venv --python 3.10
```